net.oz9aec.Gpredict.json

```diff
@@ -0,0 +1,63 @@
+{
+	"app-id": "net.oz9aec.Gpredict",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.28",
+	"sdk": "org.gnome.Sdk",
+	"command": "gpredict",
+	"finish-args": [
+		"--share=ipc", "--socket=x11",
+		"--socket=wayland",
+		"--share=network", "--persist=.config/Gpredict"
```

Comment:
You don't need `--persist` here, appropriate XDG variable will be exported and app will be write there the config automatically.